### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by dhaneshnm**
Added a few more simple test, added coverage info etc.

### CODE REVIEW

File: apps/search-poc/src/components/ResourceDetails/__tests__/__snapshots__/ResourceDeatils.test.tsx.snap [hunk 1]
```diff
@@ -2,60 +2,40 @@
 
 exports[`ResourceDetails renders correctly 1`] = `
 <div
-  className="ResourceDetails"
+  className="resource-details undefined"
```
Comment (julienmachon) [inline apps/search-poc/src/components/ResourceDetails/__tests__/__snapshots__/ResourceDeatils.test.tsx.snap]: I know you're not testing your code but when I look at that, there is obviously an issue. If not class names are passed downs as props we need to make sure we don't get `undefined` even though in reality this isn't going to break anything...

Comment (julienmachon) [reply apps/search-poc/src/components/ResourceDetails/__tests__/__snapshots__/ResourceDeatils.test.tsx.snap]: Can you maybe have a look and fix the code too in this PR?

File: apps/search-poc/src/hooks/__tests__/useMorphology.test.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}));
+
+
+
+jest.mock('../../utils/morph', () => ({
+    convertToSwc: (m) => (m),
+    parseSwc: (m) => (m),
+}));
+
+jest.setTimeout(1000);
```
[the remark this answers is not in the document]
Comment (dhaneshnm) [reply apps/search-poc/src/hooks/__tests__/useMorphology.test.tsx]: Jest has default time out, it waits until that time for the rendering part to work. This component takes more time than jest default time out, so I have set a reasonably high number for the test to pass.

File: apps/search-poc/src/hooks/useMorphology.ts [hunk 3]
```diff
@@ -56,7 +56,13 @@ export default (
         setStage('Building Render');
         viewer.addMorphology(morphology, { asPolyline: true });
         setLoading(false);
+        console.log('--------------no error-----------------');
```
[the remark this answers is not in the document]
Comment (dhaneshnm) [reply apps/search-poc/src/hooks/useMorphology.ts]: Done.

File: apps/search-poc/src/views/MainView.tsx [hunk 4]
```diff
@@ -59,6 +59,13 @@ const MainView: React.FunctionComponent<{
     [props.studioQuery],
   );
 
+  console.log('----yo ---');
```
[the remark this answers is not in the document]
Comment (dhaneshnm) [reply apps/search-poc/src/views/MainView.tsx]: Done.

File: apps/search-poc/src/components/ResourceDetails/__tests__/__snapshots__/ResourceDeatils.test.tsx.snap (hunk 1 above)
Comment (dhaneshnm) [reply apps/search-poc/src/components/ResourceDetails/__tests__/__snapshots__/ResourceDeatils.test.tsx.snap]: Updated.

File: apps/search-poc/package.json [hunk 5]
```diff
@@ -26,7 +26,7 @@
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "start": "react-scripts start",
-    "test": "react-scripts test"
+    "test": "react-scripts test --transformIgnorePatterns 'node_modules/@bbp/react-nexus/es'"
```
[the remark this answers is not in the document]
Comment (dhaneshnm) [reply apps/search-poc/package.json]: https://github.com/BlueBrain/nexus/issues/740

**Pull request merged**